gradflow.html

```diff
@@ -66,7 +66,7 @@ <h1 class="form-title">CREATE COURSE</h1>
                   <ol class="form-input-wrapper">
                      <ol class="form-input-item">
                         <label for="c-prefixnum"><b>Prefix Number</b></label>
-                        <input type="text" placeholder="Enter Prefix Number" name="c-prefixnum" id="c-prefixnum-create" pattern="^[A-Za-z]{3,4}\s?[0-9]{3}[A-Za-z]?$" title="3-4 letters, optional space, 3 digits, optional letter. ex: ABCD 123A" required>
+                        <input type="text" placeholder="Enter Prefix Number" name="c-prefixnum" id="c-prefixnum-create" pattern="^([A-Za-z]{3,4}\s?)?[0-9]{3}([A-Za-z])?$" title="optional 3-4 letters + optional space, 3 digits, optional letter. ex: ABCD 123A" required>
```

Comment:
The only thing I changed from your suggested is I am allowing the user to input without any space between major code and numbers 

For example, **ABC123** or **ABC 123** will both pass along with **123**